### PR TITLE
Change keywords

### PR DESCRIPTION
**Pull request by khiltunen**
This PR has corrected CUBA keywords.
Should fix issue #63

### CODE REVIEW

**Comment by khiltunen** [comment]
@kitchoi could you review that mayavi works now with openfoam

**Comment by kitchoi** [comment]
@khiltunen I am sorry for getting back to you so late.  
I am not sure I have all the dependencies required for testing the latest version of the poiseuille example.  I got a Segmentation Fault while running it alone.  Here is my OpenFoam situation:

I just installed OpenFoam230 but I couldn't fetch OpenFoam240 as the openfoam server is not working (Failed to fetch http://www.openfoam.org/download/ubuntu/dists/precise/main/source/Sources  404  Not Found)
I sourced the openfoam230 anyway so that my LD_LIBRARY_PATH is this:
```
/opt/ThirdParty-2.3.0/platforms/linux64Gcc/gperftools-svn/lib:/opt/paraviewopenfoam410/lib/paraview-4.1:/opt/openfoam230/platforms/linux64GccDPOpt/lib/openmpi-system:/opt/ThirdParty-2.3.0/platforms/linux64GccDPOpt/lib/openmpi-system:/usr/lib/openmpi/lib:/home/kit/OpenFOAM/kit-2.3.0/platforms/linux64GccDPOpt/lib:/opt/site/2.3.0/platforms/linux64GccDPOpt/lib:/opt/openfoam230/platforms/linux64GccDPOpt/lib:/opt/ThirdParty-2.3.0/platforms/linux64GccDPOpt/lib:/opt/openfoam230/platforms/linux64GccDPOpt/lib/dummy:/home/kit/simphony/lib
```

**Comment by khiltunen** [comment]
This is strange while travis is passing (and travis uses openfoam 2.3.0). I suppose you made installation as a user.  Are unittests passing ?

**Comment by kitchoi** [comment]
Let me try again!

**Comment by kitchoi** [comment]
@khiltunen Yes I have got it to work!

**Comment by khiltunen** [comment]
Nice!This can be closed then.
